packages/web3-eth-contract/src/AbstractContract.js

```diff
@@ -59,8 +61,10 @@ export default class AbstractContract extends AbstractWeb3Module {
         this.abiMapper = this.contractModuleFactory.createAbiMapper();
         this.options = options;
         this.PromiEvent = PromiEvent;
+        this.accounts = accounts;
         this.methodFactory = this.contractModuleFactory.createMethodFactory();
         this.abiModel = this.abiMapper.map(abi);
+        this.transactionSigner = options.transactionSigner;
         this.options = options;
```

Comment:
duplicates with line 62

Reply:
This isn't a duplication I just pass the ``transactionSigner`` dependency over the options. Check out the Eth module for seeing the real reason why I'm doing this. The complete dependency handling will get improved later (removing of the Factory objects etc.).

Reply:
I mean this assignment `this.options = options;`  it happens twice probably a left over from git merge.

Reply:
Oh sorry, I got it wrong. Thanks.